website/src/Components/Header/pageHeader.tsx

```diff
@@ -74,18 +75,31 @@ const HeaderButtons: React.FunctionComponent<{}> = () => {
   );
 };
 
-const PageHeader = () => {
+const LogoButton: React.FunctionComponent<{}> = () => {
+  // make the home button redirect to admin if admin / user profile if it is a user
   const signedInstate = useSelector(selectSignedInState);
+  let url = "/";
+  if (useSelector(selectIsAdminState)) 
```

Comment:
replace the if statement with the variable you defined above. You can't use hooks this way